File: Sprint_2/travelTest.py
import subprocess
import json

programPath = 'travelRequest.py'
countryCode = 'EG'
endPath = '/Users/joseph/Downloads/testing.txt'

#Call the program
subprocess.run(['python3', programPath, countryCode, endPath])

#Open file and print to the console
with open(endPath, 'r') as file:
    data = json.load(file)
    print(data)

File: Sprint_2/travelTest.js
const { spawnSync } = require('child_process');
const fs = require('fs');

const programPath = 'travelRequest.py';
const countryCode = 'EG';
const endPath = '/Users/joseph/Downloads/testing.txt';

//Call the proram
spawnSync('python3', [programPath, countryCode, endPath]);

//Open text file and write to console
const file = fs.readFileSync(endPath);
const data = JSON.parse(file);
console.log(data);
